stock_picking_operation_quick_change/wizards/stock_picking_wizard.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2017 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockPickingOperationWizard(models.TransientModel):
+    _name = "stock.picking.operation.wizard"
+    _description = "Stock Picking Operation Wizard"
+
+    def _prepare_default_values(self, picking):
+        return {
+            "picking_id": picking.id,
+            "location_dest_id": picking.location_dest_id.id,
+        }
+
+    @api.model
+    def default_get(self, fields):
+        res = super().default_get(fields)
+        active_model = self.env.context["active_model"]
+        active_ids = self.env.context["active_ids"] or []
+        picking = self.env[active_model].browse(active_ids)
+        res.update(self._prepare_default_values(picking))
+        return res
+
+    def _default_old_dest_location_id(self):
+        stock_picking_obj = self.env["stock.picking"]
+        pickings = stock_picking_obj.browse(self.env.context["active_ids"])
+        first_move_line = pickings.mapped("move_line_ids")[:1]
```

Comment:
No need to use mapped on relational fields, Odoo does it by default 
```suggestion
        first_move_line = pickings.move_line_ids[:1]
```